projects/igniteui-angular/schematics/tsconfig.json

```diff
@@ -4,6 +4,6 @@
         "module": "commonjs",
         "sourceMap": false,
         "declaration": true,
-        "outDir": "../../../dist/igniteui-angular/schematics/ng-add"
+        "outDir": "../../../dist/igniteui-angular/schematics/"
```

Comment:
Umm, @jackofdiamond5  Are you sure about this one?
`collection.json` still says `"factory": "./ng-add/index"`, i.e. this should break the schematic.

Reply:
The path should not lead to the `ng-add` folder as the gulp `copy-schematics` task would place the built files explicitly within the `ng-add` folder. Which basically means that if you were to add an additional schematic, to the project and then build it, it would go under `ng-add`.